pkgs/by-name/fi/fishnet/package.nix

```diff
@@ -51,12 +51,12 @@ rustPlatform.buildRustPackage (finalAttrs: {
     versionCheckHook
   ];
   doInstallCheck = true;
-  versionCheckProgram = "${placeholder "out"}/bin/${finalAttrs.meta.mainProgram}";
+  versionCheckProgram = "${placeholder "out"}/bin/fishnet";
   versionCheckProgramArg = "--version";
 
   passthru = {
     updateScript = lib.getExe (writeShellApplication {
-      name = "update-${finalAttrs.pname}";
+      name = "update-fishnet";
```

Comment:
Could you reply for https://github.com/NixOS/nixpkgs/pull/413658#discussion_r2124471219?

I think this change does not help us.